setup.py

```diff
@@ -21,6 +21,6 @@
     ],
     python_requires='>=3.7',
     install_requires=[
-        'requests==2.24.0'
+        'requests>=2.24.0'
```

Comment:
@marcelotrevisani 
Please also change requests version in `setup.cfg`
https://github.com/camunda-community-hub/camunda-external-task-client-python3/blob/ed7a493942dea6c1b570298f67b09dc12ff7da11/setup.cfg#L26